core/bubble.js

```diff
@@ -144,6 +145,17 @@ Blockly.Bubble.unbindDragEvents_ = function() {
   }
 };
 
+/*
+ * Handle a mouse-up event while dragging a bubble's border or resize handle.
+ * @param {!Event} e Mouse up event.
+ * @private
+ */
+Blockly.Bubble.bubbleMouseUp_ = function(/*e*/) {
```

Comment:
Various linters will complain about a @param with no corresponding attribute.  Either uncomment the **e**, or delete the @param and the **/\*e\*/**.